app/policies/work_policy.rb

```diff
@@ -7,8 +7,12 @@ class WorkPolicy < ApplicationPolicy
   alias_rule :delete?, to: :destroy?
 
   relation_scope :edits do |scope|
-    scope.where(depositor: user)
-         .or(scope.where(collection_id: [user.manages_collection_ids + user.reviews_collection_ids]))
+    if administrator?
```

Comment:
Should we add policy spec coverage for this or do we already have it?

Reply:
It's covered by the request spec, which is why it didn't seem necessary to have a unit test.